fecgov/fec-cms
# Update version methodology

## Summary

- Resolves #3537 
_This updates the version methodology and adds additional explainer text for unknown versions._

## Impacted areas of the application
List general components of the application that this PR will affect:

-  Independent expenditures datatable - http://localhost:8000/data/independent-expenditures/

## Screenshots

![Screen Shot 2020-02-13 at 4 05 20 PM](https://user-images.githubusercontent.com/12799132/74478048-91f5c080-4e7a-11ea-91d4-7fd0059d7ac8.png)

![Screen Shot 2020-02-13 at 4 05 26 PM](https://user-images.githubusercontent.com/12799132/74478042-8e623980-4e7a-11ea-8e96-401d3f20cbe2.png)

## How to test
- checkout this branch
- Look at the version helper text and methodology to make sure it matches what was in the #3537 ticket
____
